src/Parser/Hunter/Marksmanship/Modules/Features/CastEfficiency.js

```diff
@@ -68,7 +68,8 @@ class CastEfficiency extends CoreCastEfficiency {
     {
       spell: SPELLS.SIDEWINDERS_TALENT,
       category: CastEfficiency.SPELL_CATEGORIES.ROTATIONAL,
-      getCooldown: haste => 10.4, // TODO handle charges correctly
+      getCooldown: haste => 12/(1+haste), // TODO handle charges correctly
+      charges: 2,
```

Comment:
This handles charges already.

Reply:
I forgot to remove the TODO when I fixed it 🙃